server/controllers/appointment.js

```diff
@@ -0,0 +1,48 @@
+const asyncHandler =  require('express-async-handler');
+const validatorResults = require('express-validator');
+const Appointment = require('../models/Appointment');
+const Meeting = require('../models/Meeting');
+const moment = require('moment-timezone');
+//CREATE /appointment -> Create an appointment
+//meeting_id, name, email, time (datetime), timezone
+//GET /appointments -> list of appointments for logged in user
+
+// @route  POST api/appointment
+// @des    Creates appointment
+// @access Public
+exports.createAppointment = asyncHandler(async (req, res) => {
+    const { name, email, time } = req.body;
+    const meeting = await Meeting.findOne({ meetingId: req.params.user_id });
+    
+    const appointmentFields = {};
+    appointmentFields.meetingId = meeting;
+    if(name) appointmentFields.name = name;
+    if(email) appointmentFields.email = email;
+    if(time) appointmentFields.time = time;
+    appointmentFields.timezone = moment.tz.guess();
+
+    const newAppointment = new Appointment(appointmentFields);
+    await newAppointment.save();
+    res.json(newAppointment);
+});
+
+
+// @route  GET api/appointments
+// @des    Gets all appointments by user id
+// @access Public
+exports.getAllUserAppointments = asyncHandler(async (req, res) => {
+    try{
+        const appointments = await Appointment.findById(req.params.id);
```

Comment:
I think we should not be making the id of the meeting the same as the user, this is not a good practice in Mongo. I would instead just reference the user in the appointment model.

Reply:
So as I was saying in the comment above this one. I assumed that you were talking about the issue with making ```meetingId : req.params.user_id```? Could you explain?